guides/doc-Upgrading_and_Updating/topics/migrating_linux_7_to_linux_8.adoc

```diff
@@ -0,0 +1,44 @@
+= Migrating from RHEL 7 to RHEL 8
+
+{ProjectName} is supported on both Red Hat Enterprise Linux (RHEL) 7 and RHEL 8.
+
+Subsequent minor versions of {Project} might be supported on RHEL 8 only.
+
+If you have {ProjectName} running on RHEL 7, you must migrate to RHEL 8 to stay up-to-date on minor releases.
+
+There are two ways to migrate your server to RHEL 8:
+
+//.Unordered list
+
+* Create a backup of your server on RHEL 7, perform a fresh install of a minimal RHEL 8 instance, and restore the backup on the RHEL 8 instance, or
+* Perform an in-place migration on your server using the Leapp utility.
+
+[discrete]
+=== Fresh Install Method
+
+//.Unordered list
+
+* The fresh install migration method allows you to create a RHEL 8 system with identical specifications and the same hostname as the previous RHEL 7 system.
+
+* You can choose how much of the RHEL 7 system to include in the backup, allowing you to import only relevant parts of the RHEL 7 system to RHEL 8.
+
+* This method requires a considerable amount of system downtime as the RHEL 8 system restores the system backup; to reduce downtime, you can use the RHEL 7 system on the target machine as the backup restores.
```

Comment:
Is this implying taking the backup results in considerable downtime? Or is this implying that users will take down their RHEL 7 based installation during the restore process thus resulting in downtime?